packages/perseus-editor/src/widgets/plotter-editor.tsx

```diff
@@ -285,10 +286,16 @@ class PlotterEditor extends React.Component<Props, State> {
             this.props.type,
         );
         const canChangeSnaps = !_.contains(["pic", "dotplot"], this.props.type);
-        const props = {
-            trackInteraction: () => {},
+        const plotterProps: any = {
```

Comment:
Could we not set the type here instead of using any and then casting in the jsx?

Reply:
I honestly can't figure out any other way to avoid an error besides `ts-expect-error`, which I removed because it's worse than `any`. Open to ideas though.